makefiles/Makefile.docs-golang

```diff
@@ -0,0 +1,55 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+USER=$(shell whoami)
+
+STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
+STAGING_BUCKET=docs-mongodb-org-staging
+
+ifeq ($(SNOOTY_INTEGRATION),false)
+  PRODUCTION_URL="https://docs.mongodb.com"
+  PRODUCTION_BUCKET=docs-mongodb-org-prod
+else
+  PRODUCTION_URL="docs-mongodbcom-integration.corp.mongodb.com"
+  PRODUCTION_BUCKET=docs-go-integration
+endif
+
+PREFIX=/drivers/go
+PROJECT=golang
+MUT_PREFIX ?= $(PROJECT)
+REPO_DIR=$(shell pwd)
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+get-build-dependencies: 
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-golang.yaml > ${REPO_DIR}/published-branches.yaml
+
+next-gen-html:
+	# snooty parse and then build-front-end
+	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true"; \
+	if [ $$? -eq 1 ]; then \
+		exit 1; \
+	else \
+		exit 0; \
+	fi
+	rsync -az --exclude '.git' "${REPO_DIR}/../../snooty" "${REPO_DIR}"
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty; 
+	cd snooty; \
+	echo "GATSBY_SITE=${PROJECT}" >> .env.production; \
+	npm run build; \
+	cp -r "${REPO_DIR}/snooty/public" ${REPO_DIR};
+
+next-gen-stage: ## Host online for review
+	mut-publish public ${STAGING_BUCKET} --prefix="${MUT_PREFIX}" --stage ${ARGS};
+	echo "Hosted at ${STAGING_URL}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
+
+next-gen-deploy:	
+	if [ ${GIT_BRANCH} = main ]; then mut-redirects config/redirects -o public/.htaccess; fi
```

Comment:
@terakilobyte has updated it to master to get this out asap - so let's make that change reflected here please :)

Reply:
Oh, I think perhaps the published branches file should have been updated as well

Reply:
Oh my - I'll get that out one sec

Reply:
we're good to go https://github.com/mongodb/docs-worker-pool/blob/meta/publishedbranches/docs-golang.yaml